api/database/factories/ModelFactory.php

```diff
@@ -138,6 +138,16 @@
     ];
 });
 
+$factory->define(App\Models\Image::class, function (\Faker\Generator $faker) {
+    return [
+            'image_id' => $faker->uuid,
+            'public_id' => $faker->unique()->word,
+            'version' => time(),
+            'format' => 'png',
```

Comment:
Make this a random selection from an array of image types. There is a faker method for this - `->randomElement`